aperturedb/Utils.py

```diff
@@ -533,3 +533,52 @@ def count_descriptors_in_set(self, set_name):
             raise e
 
         return total
+
+    def remove_all_objects(self):
+
+        cmd = {"constraints": {"_uniqueid": ["!=", "0.0.0"]}}
```

Comment:
btw, what is the implication of not using this constraint ?

Reply:
It is more convenient, because you can use these constraints on the Delete* command directly. Otherwise, we would have to do a Find, declare a ref, and pass a ref to Delete*. Delete must have either a ref or constraints.